CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 1.3.2
+
+- Abandoned cart sent status is reset when customer empties cart
+- Abandoned cart status is now calculated from last item adding time instead of the first item
+- Abandoned cart activation time is used to calculate the oldest abandoned cart. Abandoned cart emails are no longer sent to very old carts.
```

Comment:
Are there any issues you can refer to with these fixes?

For example:
> Abandoned cart sent status is reset when customer empties cart [#N]

Reply:
Ok, I will add the issue numbers for all changelogs.

Reply:
Fixed https://github.com/sendsmaily/smaily-opencart-module/pull/86/commits/8c93269fea175ff3361b4f98ad380bd83281fbd8